galactic/distribution.yaml

```diff
@@ -1245,6 +1245,21 @@ repositories:
       url: https://github.com/mavlink/mavlink-gbp-release.git
       version: release/galactic/mavlink
     status: developed
+  menge_vendor:
+    doc:
+      type: git
+      url: https://github.com/open-rmf/menge_vendor.git
+      version: galactic
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ros2-gbp/menge_vendor-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/open-rmf/menge_vendor.git
+      version: master
```

Comment:
Is this right ? there is a galatic branch.

Reply:
True, fixed. Thanks for pointing out!